ajax/libs/jsrsasign/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jsrsasign",
   "version": "7.1.2",
   "description": "opensource free pure JavaScript cryptographic library supports RSA/RSAPSS/ECDSA/DSA signing/validation, ASN.1, PKCS#1/5/8 private/public key, X.509 certificate, CRL, CMS SignedData, TimeStamp and CAdES and JSON Web Signature(JWS)/Token(JWT)/Key(JWK).",
-  "filename": "jsrsasign.min.js",
+  "filename": "jsrsasign-latest-all-min.js",
```

Comment:
You should modify filename in the second commit. Otherwise, the first commit won't pass the test.